Type parameters not properly formatted

Take some error message that outputs method signatures, for example this for the Nullness Checker:

```
import org.checkerframework.checker.nullness.qual.Nullable;

class TPFormat {
  class Super {
    <A, B> void foo(A a, B b) {}
  }

  class Sub extends Super {
    <A, B extends Object> void foo(A a, B b) {}
  }
}
```

The correct error message is:

```
TPFormat.java:9: error: [override.param.invalid] <A extends @Initialized @Nullable ObjectB extends @Initialized @NonNull Object> @NonNull void foo(@Initialized @NonNull Sub this, A extends @Initialized @Nullable Object p0, B extends @Initialized @NonNull Object p1) in TPFormat.Sub cannot override <A extends @Initialized @Nullable ObjectB extends @Initialized @Nullable Object> @NonNull void foo(@Initialized @NonNull Super this, A extends @Initialized @Nullable Object p0, B extends @Initialized @Nullable Object p1) in TPFormat.Super; attempting to use an incompatible parameter type
    <A, B extends Object> void foo(A a, B b) {}
                                          ^
  found   : B extends @Initialized @NonNull Object
  required: B extends @Initialized @Nullable Object
1 error
```

However, note that in the type parameter lists there is no `, ` between `ObjectB`.
Type parameter output should properly separate the individual parameters by `, `.

--- a/framework/src/org/checkerframework/framework/type/DefaultAnnotatedTypeFormatter.java
+++ b/framework/src/org/checkerframework/framework/type/DefaultAnnotatedTypeFormatter.java
@@ -1,6 +1,7 @@
 package org.checkerframework.framework.type;
 
 import com.sun.tools.javac.code.Type;
+import java.util.ArrayList;
 import java.util.Collections;
 import java.util.IdentityHashMap;
 import java.util.List;
@@ -21,6 +22,7 @@ import org.checkerframework.framework.type.AnnotatedTypeMirror.AnnotatedWildcard
 import org.checkerframework.framework.type.visitor.AnnotatedTypeVisitor;
 import org.checkerframework.framework.util.AnnotationFormatter;
 import org.checkerframework.framework.util.DefaultAnnotationFormatter;
+import org.checkerframework.framework.util.PluginUtil;
 import org.checkerframework.javacutil.TypeAnnotationUtils;
 
 /**
@@ -252,9 +254,11 @@ public class DefaultAnnotatedTypeFormatter implements AnnotatedTypeFormatter {
             StringBuilder sb = new StringBuilder();
             if (!type.getTypeVariables().isEmpty()) {
                 sb.append('<');
+                List<String> typeVars = new ArrayList<>(type.getTypeVariables().size());
                 for (AnnotatedTypeVariable atv : type.getTypeVariables()) {
-                    sb.append(visit(atv, visiting));
+                    typeVars.add(visit(atv, visiting));
                 }
+                sb.append(PluginUtil.join(", ", typeVars));
                 sb.append("> ");
             }
             if (type.getReturnType() != null) {

